.github/workflows/cd.yml

```diff
@@ -20,11 +20,17 @@ jobs:
     - name: Checkout repo
       uses: actions/checkout@v2
     - name: Install SSH agent
-      uses: webfactory/ssh-agent@v0.4.0
+      uses: webfactory/ssh-agent@v0.4.1
```

Comment:
how do we know there is an update?
could you link the reference as comment?

Reply:
https://github.com/onflow/flow-go/commit/990126a9ea5d7dfcaba5c1ee3fc8737f09111cc6